rust/pact_ffi/src/verifier/handle.rs

```diff
@@ -83,12 +74,19 @@ impl VerifierHandle {
     port: u16,
     path: String
   ) {
+    let port = if port == 0 { None } else { Some(port) };
     self.provider = ProviderInfo {
       name,
-      protocol: scheme,
+      protocol: scheme.clone(),
       host,
-      port: if port == 0 { None } else { Some(port) },
-      path
+      port: port.clone(),
+      path: path.clone(),
+      transports: vec![ ProviderTransport {
+        transport: scheme.clone(),
+        port,
+        path: if path.is_empty() { None } else { Some(path) },
+        scheme: None
```

Comment:
If an `https` scheme as been passed to this function, why is it being set to `None` for this transport?

I believe this is causing pact-python to attempt testing a HTTPS provider via a `http://` URL

Reply:
That's a good catch, I don't think it was intentional. I need to review what `None` means here. I hope it means use the default for the protocol, but maybe that logic is not working properly.

Reply:
Confirmed the issue. If the scheme is `None`, it will use `http`.

Reply:
Corrected in efc54d26